src/libraries/ref.builds

```diff
@@ -1,6 +1,10 @@
 <Project DefaultTargets="Build" >
   <Import Project="$(MSBuildThisFileDirectory)Directory.Build.props" />
 
+  <ItemGroup Condition="'$(RuntimeFlavor)' == 'Mono'"> 
+    <ProjectExclusions Include="$(MSBuildThisFileDirectory)\System.Utf8String.Experimental\ref\System.Utf8String.Experimental.csproj" />
+  </ItemGroup>
```

Comment:
The UTF8 feature is presently specific to the CoreCLR runtime. After discussion we decided to simply exclude this project when building against the Mono runtime. This file, as well as `src.builds`, was the most convenient way for me to exclude the project from the build. If there is a more appropriate location please let me know.